css/customstyles.scss

```diff
@@ -103,11 +109,6 @@ pre, code {
   color: $cl_blue;
```

Comment:
I think hyphenation should be disabled for both `pre` and `code`. I can't see of any valid case for automatically hyphenating content that the author explicitly wanted to render preformatted.

Also, incidentally, I don't clearly see why we use a different border for `code` and `pre`.

Reply:
Agree re: de-hyphenating the `pre` blocks as well.

Re: border color: `pre` currently has no border.  Are you suggesting to give it a border similar to the `code` blocks?